ni_measurementlink_generator/ni_measurementlink_generator/template.py

```diff
@@ -104,6 +110,22 @@ def _resolve_service_class(service_class: str, display_name: str) -> str:
     "--directory-out",
     help="Output directory for measurement files. Default: '<current_directory>/<display_name>'",
 )
+@click.option(
+    "-c",
+    "--collection",
+    default="",
+    help="\b\nThe collection that this measurement belongs to. Collection names are specified"\
+        "using a period-delimited namespace hierarchy and are case-insensitive."\
+        "\nExample: 'CurrentTests.Inrush'",
+)
+@click.option(
+    "-t",
+    "--tags",
+    default=[],
+    multiple=True,
+    help="\b\nTags describing the measurement. This option may be repeated to specify multiple tags and are case-insensitive."\
```

Comment:
subject/verb agreement: "option" is singular and "are" is plural

```suggestion
    help="\b\nTags describing the measurement. This option may be repeated to specify multiple tags. Tags are case-insensitive."\
```